microsoft-edge/progressive-web-apps-chromium/whats-new/PWAs.md

```diff
@@ -0,0 +1,115 @@
+---
+description: PWA related New Features
+title: What's new in Progressive Web Apps
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 06/02/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, pwas, pwa, progressive web apps
+---
+# What's New In Progressive Web Apps
+
+[!INCLUDE [contact DevTools team note](includes/edge-whats-new-note.md)]
+
+## What's New in Edge 93
+Edge 93 became the beta channel of Microsoft Edge on August 3, 2021. This article lists updates we made to PWAs from both a developer and consumer point of view.
+
+## Window Controls Overlay Origin Trials
```

Comment:
Should be ### (H3) right?

Reply:
Well we wanted them to appear in the right hand jump to area and that's only populated by L2's so Reeza suggested we use L2's.

Reply:
That breaks the document outline which has negative implications for screen readers (in addition to causing visual hierarchy issues for sighted users). Is there another way to achieve the desired user experience in terms of the table of contents?

Reply:
Here's the broad-scoped What's New page, v93.  https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools
h1 article title: "What's New In DevTools (Microsoft Edge 92)"
The Microsoft items are flat h2's in the top half of that doc, not contained in a parent heading that says "Microsoft".  The first sentence in that article is "The following sections list the announcements from the Microsoft Edge DevTools team."
The items from Chromium are in a parent h2 "Announcements from the Chromium project", those items are h3's so don't appear in "In this article".